packages/arb-token-bridge-ui/src/components/TransactionHistory/PendingClaimsInfo.tsx

```diff
@@ -0,0 +1,63 @@
+/*
+  A small info banner that we show when the user has some pending claims.
+  Format: "You have [X] withdrawals ready to claim. [CTA]" 
+*/
+
+import { CheckCircleIcon } from '@heroicons/react/outline'
+import { OutgoingMessageState } from 'token-bridge-sdk'
+import { useNetworksAndSigners } from '../../hooks/useNetworksAndSigners'
+import { useAppState } from '../../state'
+import { isPending, outgoungStateToString } from '../../state/app/utils'
+import { isFathomNetworkName, trackEvent } from '../../util/AnalyticsUtils'
+import { getNetworkName } from '../../util/networks'
+import { useAppContextDispatch } from '../App/AppContext'
+import { ExternalLink } from '../common/ExternalLink'
+
+export const PendingClaimsInfo = () => {
```

Comment:
nit: name is a bit vague, let's rename to `WithdrawalsReadyToClaimInfo`

Reply:
Done